README.md

```diff
@@ -1,3 +1,62 @@
 # calendar-server
 
 This needs a recent enough node that has some ES6 subtleties (especially destructuring assignments).
+
+## API
+
+The API root is `/api/v1`. All following routes are relative to this root.
+
+### POST `/login`
+
+This takes 2 parameters: `user`, `password`. The input can be either
+`application/json` or `application/x-www-form-urlencoded`, with the correct
+`Content-Type` header.
+
+This returns a session ID as the cookie `connect.sid`. This cookie should be
+used in all subsequent requests, otherwise requests will get redirected to
+`/login` again.
```

Comment:
actually, I'll change this from a cookie to something not automatic  (likely returning in a json file, and requesting with a specific header), because CSRF